modules/preview-react/side-panel/stories/examples/AlwaysOpen.tsx

```diff
@@ -24,7 +19,9 @@ export const AlwaysOpen = () => {
       <SidePanel {...panelProps}>
         <Flex alignItems="center" paddingY="s" paddingX="s">
           <StyledAccentIcon icon={rocketIcon} />
-          <StyledHeader {...labelProps}>Tasks Panel</StyledHeader>
+          <TypeBodyLevel as="h3" size="large" color="licorice500" fontWeight="bold" {...labelProps}>
+            Tasks Panel
+          </TypeBodyLevel>
```

Comment:
Same thing here, maybe I'm not sure when we use `Heading` component vs `TypeBodyLevel`!